packs/vsphere/actions/get_vm.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "get_vm",
+    "runner_type": "run-python",
+    "description": "Retrieves the virtual machines on a vCenter Server system. It computes the union of Virtual Machine sets based on each parameter.",
```

Comment:
Yes, that would be nice to make it mutually-exclusive explicitly. And, the union is the second best thing for it :)